### PR TITLE
DOC-1491: Remove deprecated settings

### PR DESCRIPTION
**Pull request by tiny-ben-tran**
Related Ticket: DOC-1491

Description of Changes:
* Remove the deprecated `convert_fonts_to_spans` and `autoresize_on_init` settings

Pre-checks:
- [x] Branch prefixed with `feature/` or `hotfix/`
- [x] ~`_data/nav.yml` has been updated (if applicable)~
- [x] ~Files has been included where required (if applicable)~
- [x] Files removed have been deleted, not just excluded from the build (if applicable)
- [x] (New product features only) Release Note added

Review:
- [x] Documentation Team Lead has reviewed
- [x] Product Manager has reviewed

### CODE REVIEW

File: modules/ROOT/pages/autoresize.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 = Autoresize plugin
 
 :title_nav: Autoresize
-:description_short: :description: Automatically resize TinyMCE to fit content. :keywords: height width max_height min_height autoresize_on_init autoresize_overflow_padding autoresize_overflow_padding
+:description_short: :description: Automatically resize TinyMCE to fit content. :keywords: height width max_height min_height autoresize_overflow_padding autoresize_overflow_padding
```
Comment (lnewson) [inline modules/ROOT/pages/autoresize.adoc:4]:
```suggestion
:description_short: :description: Automatically resize TinyMCE to fit content.
:keywords: height width max_height min_height autoresize_overflow_padding autoresize_overflow_padding
```
Looks like there was some conversion issues here, so not related to your change but might as well fix it up now.

**Review by lnewson — CHANGES_REQUESTED** [review]
This has missed a number of settings listed in the Jira. At a glance I can see the following still exist:
- images_dataimg_filter
- force_hex_style_colors

File: modules/ROOT/partials/configuration/convert_fonts_to_spans.adoc [hunk 2]
```diff
@@ -1,20 +0,0 @@
-[[convert_fonts_to_spans]]
```
Comment (lnewson) [inline modules/ROOT/partials/configuration/convert_fonts_to_spans.adoc:1]: This wasn't deleted in the end (hence the strikethrough) and was delayed until 7.0 or later, so this shouldn't have been removed.

File: modules/ROOT/pages/autoresize.adoc (hunk 1 above)
Comment (tiny-ben-tran) [reply modules/ROOT/pages/autoresize.adoc:4]: Those two are removed in a [separated PR](https://github.com/tinymce/tinymce-docs/pull/2250)

Comment (lnewson) [reply modules/ROOT/pages/autoresize.adoc:4]: It should all be in a single PR...

**Review by EkimChau — APPROVED** [review]

**Review by lnewson — APPROVED** [review]

**Pull request merged**